dprint.json

```diff
@@ -0,0 +1,23 @@
+{
+  "lineWidth": 120,
+  "typescript": {
```

Comment:
See https://dprint.dev/plugins/typescript/config/ , this also works for JS, not only TS